src/screens/ChannelListScreenV2/index.tsx

```diff
@@ -53,7 +53,7 @@ const ChannelListScreenV2 = ({route}) => {
     <View>
       <StatusBar translucent={false} />
       <View style={{height: 52}}>
-        <Search route={route} onPress={goToContactScreen} />
+        <Search route={route} onPress={goToContactScreen} isShowNewChat={false} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change made here is logical and straightforward. The `isShowNewChat` prop has been added to the `Search` component with a value of `false`, which means the button for creating a new chat will not be displayed in this context. This change aligns with the PR title and description.
```diff
- <Search route={route} onPress={goToContactScreen} />
+ <Search route={route} onPress={goToContactScreen} isShowNewChat={false} />
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->